tests/kml_test.py

```
@@ -186,6 +187,79 @@
+
+        assert file_path.is_file(), "KMZ file was not created."
+
+        tree = doc.to_string()
+
+        with zipfile.ZipFile(file_path, 'r') as kmz:
+            assert 'doc.kml' in kmz.namelist(), "doc.kml not found in the KMZ file"
+
+            with kmz.open('doc.kml') as doc_kml:
+                kml_content = doc_kml.read().decode("utf-8")
+
+                assert kml_content == tree, "KML content does not match expected content"
```

Ruff

Line too long (89 > 88)

(E501)